tunnel/draft-ietf-perc-dtls-tunnel.md

```diff
@@ -407,21 +408,46 @@ distributor as soon as the HBH encryption key is computed and before
 it sends a DTLS `Finished` message to the endpoint.  The `MediaKeys`
 message includes the selected cipher (i.e. protection profile), MKI
 [@!RFC3711] value (if any), SRTP master keys, and SRTP master salt
-values. The key distributor **MUST** use the same association
+values.  The key distributor **MUST** use the same association
 identifier in the `MediaKeys` message as is used in the `TunneledDtls`
 messages for the given endpoint.
 
-The key distributor, can use the certificate of the endpoint and
-correlate that with signaling information to know which conference
-this session is associated with. The key distributor informs the media
-distributor of which conference this session is associated by sending
-a globally unique conference identifier in the `conf_id` attribute of
-the `MediaKeys`.
+The key distributor uses the certificate fingerprint of the endpoint
+along with the `tls_id` value received in the `sdp_tls_id` extension
+to determine which conference a given DTLS association is associated.
```

Comment:
If we use different ports/connections between MD and KD I think we are back to square one. Where the MD would need to know a priory which of the incoming client transport connections (ICE layer) belongs to which conference and therefore needs to be tunneled over which connection with the KD.
Therefore I think it is just a lot cleaner if we simply demand usage of 'tls_id' for this purpose.